doc/releases.md

```diff
@@ -145,9 +145,13 @@ is shown in **bold** in the index. When updating the index, please make sure
 to update the display accordingly by removing the bold styling from the previous
 release.
 
+#### Step 3: Update any REPLACEME tags in the docs
+
+If this release includes new APIs then we need to document that they were first added in this version. The relevant commits should already include REPLACEME tags as per the example in the [docs README](../tools/doc/README.md). Check for these tags with `grep REPLACEME doc/api/*.md`, and substitute this node version with `sed -i "s/REPLACEME/$VERSION/g" doc/api/*.md` or `perl -pi -e "s/REPLACEME/$VERSION/g" doc/api/*.md`.
```

Comment:
For new additions to the doc, it would be preferred to avoid uses of words like `we`, `you`, etc. For instance, the first sentence can be reworded as "If this release includes new APIs, then it is necessary to document that those were first added in this version."